Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -670,7 +670,7 @@ namespace O3DE::ProjectManager
         ResetProjectsContent();
     }
 
-    void ProjectsScreen::StartProjectDownload(const QString& projectName)
+    void ProjectsScreen::StartProjectDownload(const QString& projectName, bool queueBuild)
```

Comment:
the boolean here is a branch in your code. usually something like this can be split into two methods. you have some discretion here since this is a qt slot?

- StartProjectDownload 
- StartProjectDownloadQueued

Reply:
Having the branch here removed the need for one at the emit site and an extra Qt signal/slot.